katz/src/test/kotlin/katz/data/IorTest.kt

```diff
@@ -132,5 +133,26 @@ class IorTest : UnitSpec() {
 
         }
 
+        "Ior.monad.flatMap should be consistent with Ior#flatMap" {
+            forAll { a: Int ->
+                val implicit: Semigroup<Number> = object: Semigroup<Number> {
+                    override fun combine(a: Number, b: Number): Number = a
+                }
```

Comment:
This implicit may be flaky or different than the one in IorMonad. You could use:
```
val implicit: Semigroup<Int> = IorMonad.combine<Int>()
```
to ensure both alternatives use the same combine. Despite of that, the approach to combining probably requires more changes (see my other comments)